### ISSUE TITLE
 Missing prefix url in typescript defenitions

### ISSUE THREAD

**Issue by VsevolodTrofimov** [sindresorhus/ky#48]
ky.extend({ prefixUrl: '/api' }) throws error as prefix is missing on ky typings.
Ky version 0.4.0.

**Comment by sholladay** [comment]
Oops, sorry. I haven't been using TypeScript lately and totally forgot about that in PR #11.

I think the definition would be:

```js
/**
 * A URL to prepend to `input`. Useful when using `ky.extend()`.
 */
prefixUrl?: URL | string;
```

Looking at this, I noticed the current type definitions allow a `Request` for the `input` parameter, but they do not allow a WHATWG `URL` object. That should probably be fixed, too.

**Comment by sindresorhus** [comment]
Fixed in v0.4.1.

**Comment by sindresorhus** [comment]
> but they do not allow a WHATWG URL object. That should probably be fixed, too.

That was fixed too.
